stress/basic.go

```diff
@@ -245,7 +245,26 @@ type BasicClient struct {
 	Concurrency   int      `toml:"concurrency"`
 	SSL           bool     `toml:"ssl"`
 	Format        string   `toml:"format"`
-	addrId        int
+
+	addrId   int
+	r        chan<- response
+	interval time.Duration
+}
+
+func (c *BasicClient) retry(b []byte, backoff time.Duration) {
+	bo := backoff + time.Duration(500*time.Millisecond)
```

Comment:
OK, great, as long as you know what you are doing. 

That said, this value should be defined as a constant at the top of the file, and not exist as a magic number.